packages/playground/src/weblets/tf_deployment_list.vue

```diff
@@ -184,6 +184,20 @@
           />
         </template>
 
+        <template #Nextcloud-actions="{ item }">
+          <IconActionBtn
+            tooltip="Show Details"
+            icon="mdi-eye-outline"
+            @click="openDialog(tabs[activeTab].value, item)"
+          />
+          <IconActionBtn
+            tooltip="Open Nextcloud"
+            color="white"
+            icon="mdi-web"
+            :href="'https://' + item.value[0].publicIP?.ip.split('/')[0] + ':8080'"
```

Comment:
Can you change the URL to use a domain like it's been done in Caprover & Owncloud?

Reply:
Hi @zaelgohary. First, thanks for your feedback and review :)

As I understand, the way Nextcloud AIO works is that it asks you to log to the IP address of the server first, then it gives a password and ask the user to give the domain name, then you can select how you want Nextcloud AIO installed (e.g. if you want Nextcloud Talk, etc.) then after the installations of those packages, the user can access the domain directly.

Reply:
There might be a way to set the domain directly. I will have a look.

Reply:
@zaelgohary I did the changes you asked.
Let me know if this is OK. Thanks!